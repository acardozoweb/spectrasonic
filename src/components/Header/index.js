////////////////////////
///// HEADER INDEX /////
////////////////////////

import React from 'react';
import spectraLogo from '../../assets/logos/spectrasonic_logo_k-192.jpg'

function Header() {
    return (
        <section>
            <img src={spectraLogo} className="spectraLogo" style={{ width: "35%" }} alt="spectrasonic logo"/>
        </section>
    );
}

export default Header;

