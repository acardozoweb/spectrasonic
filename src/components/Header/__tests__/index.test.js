import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Header from "..";

afterEach(cleanup);

describe("About component", () => {
  // First Test
  it("renders", () => {
    render(<Header />);
  });

  // Second Test
  it('matches snapshot DOM node structure', () => {
      const { asFragment } = render(<Header />);
      expect(asFragment()).toMatchSnapshot();
  })
});
