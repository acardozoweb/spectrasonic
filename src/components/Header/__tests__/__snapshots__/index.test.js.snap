// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section>
    <img
      alt="spectrasonic logo"
      class="spectraLogo"
      src="spectrasonic_logo_k-192.jpg"
      style="width: 35%;"
    />
  </section>
</DocumentFragment>
`;
